### PR TITLE
fix triple-servlet-dependencies

### PR DESCRIPTION
**Pull request by heliang666s**
## What is the purpose of the change
fix triple-servlet-dependencies.
图片
图片

## Brief changelog
add related dependencies in pom.xml

## Verifying this change
No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Check if is necessary to patch to Dubbo 3 if you are work on Dubbo 2.7
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Add some description to [dubbo-website](https://github.com/apache/dubbo-website) project if you are requesting to add a feature.
- [ ] GitHub Actions works fine on your own branch.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Review by oxsean — CHANGES_REQUESTED** [review]
I don't think it's necessary to make these changes because the files are already shaded to the dubbo.jar, please check whether dubbo.jar is correctly packaged, adding it is acceptable to avoid idea compilation problems, but it needs to set
 ```
<optional>true</optional>
```

File: dubbo-spring-boot/dubbo-spring-boot-3-autoconfigure/pom.xml [hunk 1]
```diff
@@ -65,6 +65,19 @@
...
+      <groupId>org.apache.dubbo</groupId>
+      <artifactId>dubbo-rpc-triple</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.dubbo</groupId>
+      <artifactId>dubbo-triple-servlet</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```
Comment (oxsean) [inline dubbo-spring-boot/dubbo-spring-boot-3-autoconfigure/pom.xml:81]:
```
<optional>true</optional>
```

File: dubbo-spring-boot/dubbo-spring-boot-autoconfigure/pom.xml [hunk 2]
```diff
@@ -77,6 +77,18 @@
...
+      <groupId>org.apache.dubbo</groupId>
+      <artifactId>dubbo-rpc-triple</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.dubbo</groupId>
+      <artifactId>dubbo-triple-servlet</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```
Comment (oxsean) [inline dubbo-spring-boot/dubbo-spring-boot-autoconfigure/pom.xml:92]:
```
<optional>true</optional>
```

File: dubbo-spring-boot/dubbo-spring-boot-autoconfigure/pom.xml [hunk 3]
```diff
@@ -77,6 +77,18 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.dubbo</groupId>
+      <artifactId>dubbo-rpc-triple</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```
Comment (oxsean) [inline dubbo-spring-boot/dubbo-spring-boot-autoconfigure/pom.xml:85]:
```
<optional>true</optional>
```

**Review by AlbumenJ — APPROVED** [review]

**Pull request merged**